### ISSUE TITLE
I2C Slave with USB 

### ISSUE THREAD

**Issue by dontgetfoundout** [Xinyuan-LilyGO/T-Display-S3-Long#17]
TL;DR: Can the USB and powered QWIIC connectors be used at the same time?

Hello,
  I'm working on a project that uses the T-Display Long as an I2C slave through the QWIIC connector, controlled by another microcontroller board.  The T-Display appears to be able to power itself from the 3.3V from the QWIIC connector.  While developing the project, it would be useful to be able to also connect the board to USB for flashing/debugging/serial output.  I'm wondering if it's possible to connect both a QWIIC connector supplying 3.3V and USB (@ 5V) at the same time, but I don't want to damage either board in the process.  Is it safe to connect both at the same time?  Are there any risks to consider?

**Comment by dontgetfoundout** [comment]
Seems to work okay
